style.css

```diff
@@ -13,170 +13,479 @@
+  .header {
+    padding: 0 50px;
   }
 
-  .nav-item-action {
-    font-size: 15px;
-    font-weight: 500;
-    color: #344563;
-    line-height: 20px;
-    padding: 10px 12px;
-    text-decoration: none;
+  .header-manu {
+    display: none;
   }
 
-  .nav-item-action:hover {
-    color: #fff;
-    background-color: #344563 !important;
+  .nav-items {
+    display: flex;
+    align-self: center;
+    justify-self: center;
+    flex-direction: row;
   }
 
   .mobile-nav-items {
-    top: 0;
-    left: 0;
-    bottom: 0;
-    width: 100%;
-    height: 100vh;
     display: none;
   }
 
   .article {
+    background-image: url("./resources/images/header-shapes\ Deasktop\ BG.png");
     display: flex;
-    min-height: 100vh;
+    min-height: 113vh;
     flex-direction: column;
-    background-color: #fff;
-    border-bottom-left-radius: 70px;
+    background-color: white;
+    border-bottom-left-radius: 15%;
     background-repeat: no-repeat;
-    background-size: contain;
-    background-position: center;
-    background-image: url("./resources/images/header-shapes\ mobilebackground-Main.png");
+    background-size: 100%;
+    background-position: 99% 25%;
   }
 
   .article-item1 {
-    margin: 20% 12% 15%;
+    align-self: center;
+    max-width: 920px;
   }
 
   .headline-title {
     color: #172b4d;
     font-weight: 900;
     font-size: 40px;
-    font-style: bolder;
     line-height: 52px;
     display: flex;
     flex: none;
-    margin-bottom: 20px;
     font-family: Poppins-bold, sans-serif;
   }
 
+  .headline-container {
+    align-self: center;
+    justify-self: center;
+    max-width: 100vw;
+  }
+
   .headline-content {
-    font-size: 16px;
+    font-size: 20px;
     color: #344563;
     font-style: normal;
     font-weight: 400;
-    line-height: 30px;
-    flex: none;
-    margin-left: 0;
-    margin-right: 24px;
-  }
-
-  .headline-container {
-    display: grid;
-    row-gap: 25px;
-  }
-
-  .social-links {
-    display: flex;
-    list-style: none;
-    margin-top: 10px;
-    padding-left: 1%;
-    padding-right: 10px;
+    line-height: 35px;
+    margin-right: 25%;
   }
 
   .social-title {
-    font-size: 16px;
+    font-size: 20px;
     font-weight: 600;
     text-decoration: none;
     color: #7f8cff;
-    line-height: 50px;
-    width: 120px;
-    height: 24px;
+    line-height: 60px;
   }
 
   .work-section {
-    margin: 80px 0;
-    display: grid;
-    row-gap: 80px;
-    padding: 10px;
-    justify-items: center;
+    margin: 100px;
   }
 
   .work-card {
-    background: #fff;
-    border-radius: 10px;
-    min-height: 3em;
-    border: 1px solid #c6c6c6;
-    padding: 20px 15px;
-    display: grid;
+    display: flex;
+    flex-direction: row;
   }
 
   .work-preview {
-    display: flex;
-    align-self: center;
-    flex-direction: column;
-    justify-self: center;
-    overflow: hidden;
+    padding-left: 3%;
+    height: 500px;
+    width: 50%;
+    justify-content: center;
   }
```

Comment:
- [ ] Please make sure that your design looks as close to the `Figma` design as possible. Looking at your work card I can see you have `empty` space at the **top** and **bottom** of the card. This is because you have specified a `height` of `50%` for the card. To fix this issue so your design looks like that of the `Figma` design and also help create a good user experience please consider the following hints:
 `HINT:` Remove the `height: 50%` and increase the `width` to `100%`. This will make your image cover the full width of its container. See below for more details:
  <table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="1170" alt="Screenshot 2023-06-14 at 20 32 54" src="https://github.com/Uzair-Manzoor/Portfolio/assets/107192076/b8041807-ee6e-47e3-8254-057581263a4c">
</td>
<td>
<img width="1030" alt="Screenshot 2023-06-14 at 20 31 37" src="https://github.com/Uzair-Manzoor/Portfolio/assets/107192076/ee1eac0d-6a4b-49e6-ab54-0e19016e8b5c">
</td>
</tr>
</table>